### ISSUE TITLE
Changing genetical UI+UE vs flavour text ooc/ic issue

### ISSUE THREAD

**Issue by Deelite34** [ParadiseSS13/Paradise#10404]
**Problem Description**:
What is the problem?
When you change your ui, into ui of other player, you keep your old flavour text. 
**What did you expect to happen**: 
Why do you think this is an issue?
Using not-changing flavour text to find out the sneaky beaky like antags

**What happened instead**:
How is what happened different from what you expected?
Either flavour text should be removed when you change identity using ui, or change it onto target's flavour text.

**Why is this bad/What are the consequences:**
Why do you think this is an important issue?
Example situation that could happen:
Hmm, mister Holden Price, since when you start to look so mexican? I though mister Muchachos was the only mexican on the station? I guess I will search you thoroughly, and make sure to find a ic proof that you are an vampire, like one we had found out earlier in shift. Meanwhile, please stay cuffed-buckled in processing for an hour.

![wrong ui](https://user-images.githubusercontent.com/35972878/49435865-e7830780-f7b7-11e8-8b81-4db5cf9d3a8e.png)

**Steps to reproduce the problem**:
The most important section. Review everything you did leading up to causing the issue.
Get someones UI+UE syringe using genetics console, and inject yourself. You keep your flavour text.

**Comment by Scout360Pyro** [comment]
Playing Devils advocate here. The way I see flavor text typically used is to add flavor by describing body language, facial expressions, maybe a small but IC easily noticeable habit like humming, etc. Things more detailed around personality than what your DNA would hold.

One could argue that someone impersonating you would not hold those habits, and thus looking at the flavor text and remembering that it is SUPPOSED to be something else would be like noticing the person is acting strange. Granted the example you used of "looking mexican" doesn't hold very well to this, but I believe it makes sense for most flavor texts.

Ultimately I think this is something the Admins/Devs should figure out whether to define as a Bug or a Feature that guidelines may be needed for.

**Comment by Mitchs98** [comment]
I pretty much agree with the above statement, honestly. Though that aside I'd consider it potentially a metagaming thing as well if that was the sole basis for everything.

**Comment by Ar3nn** [comment]
nooooooooooo a thousand times no. There are already enough powergamers that go around shift-clicking everyone in sight. I disagree entirely that this should be a feature at all.

**Comment by Mitchs98** [comment]
> nooooooooooo a thousand times no. There are already enough powergamers that go around shift-clicking everyone in sight. I disagree entirely that this should be a feature at all.

How is shift clicking everyone power gaming? You stare at people irl, there is literally no difference..

**Comment by Ar3nn** [comment]
Yes, but you don't intensively study everyone you see every second of every day. That would be tiresome. Looking at someone, you see what they look like and not much else, unless you take time to study them, which is what shift-clicking represents. If you just want to see someone, you see them on your screen, which takes care of seeing them at the surface level.

**Comment by taukausanake** [comment]
Wow. Am I an irl powergamer if I actually study others around me? 😕

**Comment by Mitchs98** [comment]
> Yes, but you don't intensively study everyone you see every second of every day. That would be tiresome. Looking at someone, you see what they look like and not much else, unless you take time to study them, which is what shift-clicking represents. If you just want to see someone, you see them on your screen, which takes care of seeing them at the surface level.

Not true whatsoever considering glancing at them on your screen just gives a generic spaceman. You can't really tell anything about them just glancing at them. That's the whole point of flavor text...

**Comment by Migratingcocofruit** [comment]
fixed by #26094
